jsonnet/kube-prometheus/alertmanager/alertmanager.libsonnet

```diff
@@ -40,6 +40,10 @@ local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
         ],
       },
       replicas: 3,
+      resources: {
+        limits: { cpu: '100m', memory: '100Mi' },
+        requests: { cpu: '100m', memory: '100Mi' },
```

Comment:
Why limits and requests are the same?

Reply:
I would guess to make sure that we can schedule and always have these resources available.